app/main.py

```
@@ -349,73 +341,105 @@ def update_jdk_metrics():
-                BUILD_STATUS_GAUGE.labels(
-                    template_id=template_id,
-                    build_type_name=cfg["name"],
-                    build_type_id=cfg["id"],
-                    build_url=cfg["webUrl"]
-                ).set(status_value)
-
-        logging.info(f"Build status metrics updated successfully")
-
+                _set_build_metrics(template_id, cfg, last_build)
+        logging.info("Build status metrics updated successfully")
     except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)